build.rs

```diff
@@ -286,6 +286,18 @@ fn test_parse_nasm_version() {
   assert_eq!((2, 14, 0), parse_nasm_version(ver_str));
 }
 
+#[cfg(feature = "asm")]
+fn is_nasm_installed_check() {
+  let mut cmd = std::process::Command::new("nasm");
+  cmd.arg("-h");
+  match cmd.output() {
+    Ok(_) => (),
+    Err(_) => {
+      panic!("Couldn't find NASM, make sure NASM installed and in PATH")
```

Comment:
Might be good to mention that is possible to not set the `asm` feature to avoid installing it.

Reply:
Corrected